config/abilities/master_admin.rb

```diff
@@ -64,6 +64,8 @@
     can :manage, Invitation
     can :manage, Invoice, provider_account_id: user.account_id unless user.account.master_on_premises?
 
+    can :manage, :permissions
```

Comment:
Added to be able to detach "member permissions" from the provider's "groups" switch. See https://github.com/3scale/porta/pull/2243/files#r518718702.